modules/investigating-worker-node-installation-issues.adoc

```diff
@@ -0,0 +1,200 @@
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* You have SSH access to your hosts.
+* You have the fully qualified domain names of the bootstrap and worker nodes.
+* If you are hosting Ignition configuration files by using an HTTP server, you must have the HTTP server's fully qualified domain name and the port number. You must also have SSH access to the HTTP host.
++
+[NOTE]
+====
+The initial `kubeadmin` password can be found in `<install_directory>/auth/kubeadmin-password` on the installation host.
+====
+
+.Procedure
+
+. If you have access to the worker node's console, monitor the console until the node reaches the login prompt. During the installation, Ignition log messages are output to the console.
+
+. Verify Ignition file configuration.
++
+* If you are hosting Ignition configuration files by using an HTTP server.
++
+.. Verify the worker node Ignition file URL. Replace `<http_server_fqdn>` with HTTP server's fully qualified domain name:
++
+----
+curl -I http://<http_server_fqdn>:<port>/worker.ign  <1>
+----
+<1> The `-I` option returns the header only. If the Ignition file is available on the specified URL, the command returns `200 OK` status. If it is not available, the command returns `404 file not found`.
++
+.. To verify that the ignition file was received by the worker node, query the HTTP server logs on the HTTP host. For example, if you are using an Apache web server to serve Ignition files:
++
+----
+$ grep -is 'worker.ign' /var/log/httpd/access_log
+----
++
+If the worker Ignition file is received, the associated `HTTP GET` log message will include a `200 OK` success status, indicating that the request succeeded.
++
+.. If the Ignition file was not received, check that it exists on the serving host directly. Ensure that the appropriate file and web server permissions are in place.
++
+* If you are using a cloud provider mechanism to inject Ignition configuration files into hosts as part of their initial deployment.
++
+.. Review the worker node's console to determine if the mechanism is injecting the worker node Ignition file correctly.
+
+. Check the availability of the worker node's assigned storage device.
+
+. Verify that the worker node has been assigned an IP address from the DHCP server.
+
+. Determine worker node status.
+.. Query node status:
++
+----
+$ oc get nodes
+----
++
+.. Retrieve a detailed node description for any worker nodes not showing a `Ready` status:
++
+----
+$ oc describe node <worker_node>
+----
++
+[NOTE]
+====
+It is not possible to run `oc` commands if an installation issue prevents the {product-title} API from running or if the kubelet is not running yet on each node:
```

Comment:
I think the sentence should end with a period.

Reply:
Resolved.